src/selfie_launch/launch/parameters.yaml

```diff
@@ -0,0 +1,125 @@
+const_setpoint: 
+  {
+    topic: setpoint,
+    value: -0.2
+  }
+detect_markings: 
+  {  
+    debug_mode: false,
+    pf_num_points: 3,
+    pf_num_samples: 50,
+    pf_num_samples_vis: 4,
+    pf_std: 0.015,
+    real_window_size: 0.1,
+    threshold_c: -15
+  }
+detect_parking_spot: 
+  {
+    default_speed_in_parking_zone: 0.8,
+    max_distance_to_free_place: 0.8,
+    point_max_x: 2,
+    point_max_y: 0.2,
+    point_min_x: 0.01,
+    point_min_y: -1,
+    speed_when_found_place: 0.3,
+    visualization_in_searching: false
+  }
+extract_road_features: 
+  {
+    average_window_size: 10,
+    lookahead: 0.55,
+    max_acceleration: 1.0,
+    max_curvature: 1.8,
+    max_deceleration: 1.5,
+    min_speed: 1.5
+  }
+free_drive: 
+  {
+    max_speed: 3.0,
+    starting_line_distance_to_end: 0.45
+  }
+lidar: 
+  {
+    serial_port: /dev/sensors/hokuyo_H1312790
```

Comment:
Na pewno to działa? Nie powinno być w "" jako string?

Reply:
Według specyfikacji YAML w [rosie](http://wiki.ros.org/YAML%20Overview) i samej specyfikacji yamlowej, stringi muszą być objęte "" lub '' tylko wtedy, gdy ciąg znaków posiada w sobie hasz poprzedzony spacją ' #'(przez YAML traktowany jako komentarz), spację poprzedzoną dwukropkiem ": " lub znaki specjalne typu "\n \t (...)" itd. Wszystkie napisy bez tych znaków traktowane są jako string, a ciągi znaków jako stało lub zmiennoprzecinkowe liczby, ale dla ścisłości zmienię.